lib/notifications/EmailNotifier.ts

```diff
@@ -0,0 +1,336 @@
+    }
+  };
+
+  private getMinerFeeSymbol = (symbol: string): string => {
+    const currency = this.service.currencies.get(symbol);
+    if (!currency) return symbol; // Fallback
+
+    if (currency.type === CurrencyType.ERC20) {
+      return (
+        this.walletManager.ethereumManagers.find((manager) =>
+          manager.hasSymbol(symbol),
+        )?.networkDetails.symbol || symbol // Fallback to original symbol
+      );
+    } else {
+      return symbol;
+    }
+  };
+
+  private sendSuccessEmail = async (
+    swapDetails: AnySwap,
+    channelCreation?: ChannelCreation,
+  ): Promise<void> => {
+    if (!this.transporter || !this.config) {
+      return;
+    }
+
+    const { base, quote } = splitPairId(swapDetails.pair);
+    const { sending, receiving } = getSendingReceivingCurrency(
+      base,
+      quote,
+      swapDetails.orderSide,
+    );
+
+    const hasChannelCreation =
+      channelCreation !== null &&
+      channelCreation !== undefined &&
+      channelCreation.fundingTransactionId !== null;
+
+    const subjectPrefix = this.config.subjectPrefix || '[Boltz]';
+    const title = `Swap ${this.getSwapTitle(swapDetails.type, sending, receiving)}${hasChannelCreation ? ` 🏗️` : ''}`;
+    const subject = `${subjectPrefix} ${title}: ${swapDetails.id}`;
+
+    let body =
+      `${title}\n\n` +
+      `${await this.getBasicSwapInfo(swapDetails.type, swapDetails, base, quote)}\n` +
+      `Fees earned: ${satoshisToSatcomma(swapDetails.fee!)} ${sending}\n` + // Assuming fee is always in sending currency
+      `Miner fees: ${this.getMinerFees(swapDetails.type, swapDetails)}`;
+
+    if (swapDetails.type === SwapType.Submarine) {
+      const submarineSwap = swapDetails as Swap;
+      // The routing fees are denominated in millisatoshi
+      body += `\nRouting fees: ${
+        submarineSwap.routingFee ? submarineSwap.routingFee / 1000 : 'N/A'
+      } sats`;
+    }
+
+    if (hasChannelCreation) {
+      body +=
+        '\n\nChannel Creation:\n' +
+        `Private: ${channelCreation!.private}\n` +
+        `Inbound: ${channelCreation!.inboundLiquidity}%\n` +
+        `Node: ${channelCreation!.nodePublicKey}\n` +
+        `Funding: ${channelCreation!.fundingTransactionId}:${
+          channelCreation!.fundingTransactionVout
+        }`;
+    }
+
+    const mailOptions = {
+      from: this.config.from,
+      to: this.config.to,
+      subject: subject,
+      text: `${body}${EmailNotifier.trailingWhitespace}`,
+    };
+
+    try {
+      const info = await this.transporter.sendMail(mailOptions);
+      this.logger.info(
+        `Success email sent for swap ${swapDetails.id}: ${info.messageId}`,
+      );
+    } catch (error) {
+      this.logger.error(
+        `Error sending success email for swap ${swapDetails.id}: ${formatError(
+          error,
+        )}`,
+      );
+    }
```

Comment:
```suggestion
      const info = await this.transporter.sendMail(mailOptions);
      this.logger.info(
        `Success email sent for swap ${swapDetails.id}: ${info.messageId}`,
      );
```

Errors are already handled in the function that calls this one